assets/js/googlesitekit/data/utils.js

```diff
@@ -338,7 +338,7 @@ const findDuplicates = ( array ) => {
  * @return {Object} Object with common actions, controls and reducer.
  */
 export const commonStore = {
-	actions: addInitializeAction( commonActions ),
+	actions: commonActions,
 	controls: commonControls,
-	reducer: addInitializeReducer( {}, passthroughReducer ),
+	reducer: passthroughReducer,
```

Comment:
We strictly don't need this now, since if `reducer` is `undefined` the `combineStores` utility will fill it with that anyway. But not a big deal, so 👍